app/presenters/coronavirus_landing_page_presenter.rb

```diff
@@ -3,7 +3,6 @@ class CoronavirusLandingPagePresenter
     header_section
     risk_level
```

Comment:
`risk_level` was rendered as part of the announcements section. I think it should have been removed along with the announcements code. Can it be in this PR?

Reply:
Argh! Probably my fault sorry - fixed up